### PR TITLE
Set default values from new_edges and new_features metrics.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

File: src/python/bot/fuzzers/libFuzzer/stats.py [hunk 1]
```diff
@@ -176,6 +176,8 @@ def parse_performance_features(log_lines, strategies, arguments):
       'max_len': 0,
       'manual_dict_size': 0,
       'merge_edge_coverage': 0,
+      'new_edges': None,
```
Comment (oliverchang) [inline src/python/bot/fuzzers/libFuzzer/stats.py]: let's actually set these to 0. Having these as "None" could break new projects.

**Comment by inferno-chromium** [comment]
Discussed offline, schema autodetect does not work with None value. So, this will break new projects.

**Pull request merged**